src/main.js

```diff
@@ -42,20 +42,30 @@ const router = new VueRouter({
 
 // Route Guard rules for directing users
```

Comment:
strangers = pages only logged out people can use
friends = pages that only logged in people can use